src/gpodder/gtkui/desktop/preferences.py

```diff
@@ -317,6 +336,8 @@ def new(self):
                                               self.checkbutton_delete_deleted_episodes)
         self._config.connect_gtk_togglebutton('device_sync.compare_episode_filesize',
                                               self.checkbutton_compare_episode_filesize)
+        self._config.connect_gtk_togglebutton('network.use_proxy',
```

Comment:
All of the proxy code should be grouped together, instead of spread out over the function.